client/lang/en.js

```diff
@@ -29,6 +29,16 @@ if (typeof(ss) === 'undefined' || typeof(ss.i18n) === 'undefined') {
     "Admin.LINKLABEL_EXTERNALURL": "Link to external URL",
     "Admin.LINKLABEL_EMAIL": "Link to email address",
     "Admin.LINK_EXTERNAL": "Insert external link",
-    "Admin.LINK_EMAIL": "Insert email link"
+    "Admin.LINK_EMAIL": "Insert email link",
+    "Admin.WRITTEN_NUMBER_ZERO": "zero",
```

Comment:
So you'll find a list of translations (with code surrounding this list) named `client/lang/en.js`
You'll also find another file named `client/lang/src/en.json` which contains the same list of translations (but without the code surrounding the list)